spec/statement.md

```diff
@@ -0,0 +1,176 @@
+character`] between a new line and any non-whitespace characters of the last
+consumed statement. Lines that only contain whitespace or do not start a
+statement do not impact the `current indentation level`.
+
+A document parsed in the indented syntax may not begin with whitespace, and the
+initial `current indentation level` for any document is `0`.
+
+> Changes in the indentation level are used by the indented syntax to start and
+> end blocks of statements.
+
+[`documentation indentation character`]: #document-indentation-character
+
+### Document indentation character
+
+The `document indentation character` is defined as the first tab or space
+character used in an [`IndentMore`] production in a document.
+
+[`IndentMore`]: #indentation
+
+> The `document indentation character` is the character used for calculating the
+> [`current indentation level`]. In the indented syntax, no character other than
+> the `document indentation character` may be used for indentation.
+
+[`current indentation level`]: #current-indentation-level
+
+## Syntax
+
+### ScssStatements
+
+<x><pre>
+**ScssStatements**      ::= (Statement ';'?¹)* Statement?
+</pre></x>
+
+1. This production is mandatory unless the previous `Statement` is a
+`LoudComment`, `SilentComment`, or ends in a `Block`.
+
+If a `WhitespaceComment` would be ambiguous with a `Statement` in the
+`ScssStatements` rule, parse it preferentially as a `Statement`.
+
+### IndentedStatements
+
+<x><pre>
+**IndentedStatements**  ::= (Statement [IndentSame])* Statement
+</pre></x>
+
+[IndentSame]: #indentation
+
+The `Statement` productions may not include newlines outside of `IndentSame`
+productions.
+
+### Stylesheet
+
+<x><pre>
+**Stylesheet**          ::= U+FEFF? ([ScssStatements] | [IndentedStatements])¹
+</pre></x>
+
+[ScssStatements]: #scssstatements
+[IndentedStatements]: #indentedstatements
+
+1. Only the production for the current syntax is valid.
+
+### Block
+
+<x><pre>
+**ScssBlock**      ::= '{' [ScssStatements] '}'
+**IndentedBlock**  ::= [IndentMore] [IndentedStatements]
+**Block**          ::= (ScssBlock | IndentedBlock)¹
+</pre></x>
+
+[IndentMore]: #indentation
+
+1. Only the production for the current syntax is valid.
+
+### Comments
+
+#### LoudComment
+
+<x><pre>
+**ScssLoudComment**          ::= '/\*' (.\*¹ | Interpolation)\* '\*/'
+**IndentedLoudCommentNest**  ::= [IndentMore] ((.\*² | Interpolation)\*
+&#32;                            [IndentSame])\* (.\*² | Interpolation)\*
+**IndentedLoudComment**      ::= '/\*' (.\*² | Interpolation)\*
+&#32;                            IndentedLoudCommentNest?
+**LoudComment**              ::= (ScssLoudComment | IndentedLoudComment)³
+</pre></x>
+
+1. This may not contain `#{` or `*/`.
+2. This may not contain `#{` or newlines.
+3. Only the production for the current syntax is valid.
+
+#### SilentComment
+
+<x><pre>
+**ScssSilentComment**          ::= '//' .\*¹
+**IndentedSilentComment**      ::= '//' .\*²
```

Comment:
This can also have text nested within it.

Reply:
Added in 22c1607cb6494b98dd6aeccf8da9b877234439f3